src/Algebra/Bundles.agda

```diff
@@ -11,6 +11,7 @@
 
 module Algebra.Bundles where
 
+open import Algebra.Bundles.Raw public
```

Comment:
We usually move `public` imports out of the main import list and into their own commented section of the file to draw attention to them. e.g.

https://github.com/agda/agda-stdlib/blob/bc9c1b6a117fcb86b321113c0958ffc3b3526b4e/src/Text/Regex.agda#L26-L30

Any chance you could change this and other ones?